server/core/src/main/scala/org/infinispan/server/core/transport/StatsChannelHandler.java

```diff
@@ -0,0 +1,68 @@
+package org.infinispan.server.core.transport;
```

Comment:
The reason I did so was because the whole static variable thing in scala is not a very nice solution, imo.  I have another handler that I will be adding that will use these variables.